nodeinstaller/node-installer.go

```diff
@@ -202,21 +186,38 @@ func containerdRuntimeConfig(basePath, configPath string, platform platforms.Pla
 func patchContainerdConfig(runtimeHandler, basePath, configPath string, platform platforms.Platform) error {
 	existingRaw, existing, err := parseExistingContainerdConfig(configPath)
 	if err != nil {
+		fmt.Printf("Failed to parse existing containerd config: %v\n", err)
+		fmt.Println("Creating a new containerd base config.")
 		existing = constants.ContainerdBaseConfig()
 	}
 
-	snapshotterName := "no-snapshotter"
-	// Add tardev snapshotter, only required for AKS
-	if platform == platforms.AKSCloudHypervisorSNP {
-		if existing.ProxyPlugins == nil {
-			existing.ProxyPlugins = make(map[string]config.ProxyPlugin)
-		}
+	// Enable containerd debug logging.
+	existing.Debug.Level = "debug"
```

Comment:
Should we keep debug logging off on AKS?

Reply:
I've changed this so it is only enabled for debug runtimes on all platforms.

Reply:
Nice, thanks.